groups/storage.yml

```diff
@@ -0,0 +1 @@
+name: Storage
```

Comment:
At some point we'll need a description for these. Would you mind leaving a comment with a draft description, so we know what you were thinking when we come back to this?

Reply:
Added description